### ISSUE TITLE
How to run evaluation on the validation set?

### ISSUE THREAD

**Issue by anirudh-chakravarthy** [OpenDriveLab/DriveLM#131]
Hi,

Is is possible to provide a set of instructions to run evaluation on the validation set? 

From the README, I did the following:

> [test_eval.json](https://github.com/OpenDriveLab/DriveLM/blob/main/challenge/test_eval.json) is used for evaluation. [test_llama.json](https://github.com/OpenDriveLab/DriveLM/blob/main/challenge/test_llama.json) is used for training

However, when I run:
```
python demo.py --llama_dir /path/to/llama_model_weights --checkpoint /path/to/pre-trained/checkpoint.pth --data ../test_llama.json  --output ../output.json --batch_size 4 --num_processes 8
```

And then:
```
python evaluation.py --root_path1 ./output.json --root_path2 ./test_eval.json
```

I face [this issue](https://github.com/OpenDriveLab/DriveLM/tree/main/challenge#i-encountered-keyerror-b789de07180846cc972118ee6d1fb027_b0e6fd5561454b2789c853e5350557a8_0-in-my-submission-comment-what-should-i-do) about the random UUID. I follow the exact instructions, so I'm not sure why this doesn't work.

For further diagnosis, following the FAQ which say I should run inference on the validation set, I ran:
```
python convert2llama.py
```
and changed [this line](https://github.com/OpenDriveLab/DriveLM/blob/main/challenge/convert2llama.py#L45) to `v1_1_val_nus_q_only.json` and output to `val_llama.json`

python demo.py --llama_dir /path/to/llama_model_weights --checkpoint /path/to/pre-trained/checkpoint.pth --data ../val_llama.json  --output ../output_val.json --batch_size 4 --num_processes 8

python evaluation.py --root_path1 ./output_val.json --root_path2 ./v1_1_val_nus_q_only.json
```

But this doesn't work either, and shows the same UUID error.

**Comment by ChonghaoSima** [comment]
Could you post the UUID error here? Are you running eval on your local env or our test server?
